src/ripple/protocol/impl/InnerObjectFormats.cpp

```diff
@@ -28,6 +28,7 @@ InnerObjectFormats::InnerObjectFormats()
         {
             {sfAccount, soeREQUIRED},
             {sfSignerWeight, soeREQUIRED},
+            {sfWalletLocator, soeOPTIONAL},
```

Comment:
I understand why you want this field and concede that it's useful, but it may be slightly controversial in terms of size.

So is a 256-bit value _really_ needed? Could we get away with something smaller? I think your original suggestion was to use a 64-bit tag. I suspect that would be fine, but maybe a little too constrained for what you want to do. 128 bits is probably sufficient, especially if you use it as an [RFC-4122 UUID](https://datatracker.ietf.org/doc/html/rfc4122) (likely V1, V4 or V5, depending on the usecase).

I am not _overly_ concerned and I think it's fine as is, but I do think it's worth at least having a discussion about this to ensure the field is "right-sized".

Reply:
Reusing an existing (essentially unused) serialized field is great because it makes it backwards compatible with old codecs. So if someone proposes to change the size of the tag they should also propose an existing sfField that matches their proposed size.

Reply:
In my opinion reusing a pre-existing field is, in this case, probably a false economy.  This commit will be bound together with many others in a release.  At least one of the other commits in the release will likely change the codec.  So having this particular commit avoid changing the codec does not prevent the codec from changing in a release.

Most programmers agree that names are hard.  Giving a field the wrong name or a misleading name will only make the XRP Ledger even harder to use correctly.

My personal opinion is that you should pick the right sized `SField` type and select a truly meaningful name for the field.  Proceed with that.  If you are truly sold that `WalletLocator` is the best name you can come up with then I'd love to hear that explanation.  Thanks.